compiler+runtime/src/cpp/jank/util/string_builder.cpp

```diff
@@ -1,6 +1,9 @@
 #include <bit>
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
 #include <codecvt>
 #include <locale>
+#pragma clang diagnostic pop
```

Comment:
Is the entire header deprecated? I thought it was only `wstring_convert`.

Reply:
had to add those around the header as well. otherwise still got the complains.